.ci/update-beats.yml

```
@@ -0,0 +1,55 @@
+---
+name: Bump beats
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: apm-server
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: '{{ requiredEnv "BRANCH_NAME" }}'
+
+actions:
+  default:
+    kind: github/pullrequest
+    scmid: default
+    spec:
+      automerge: false
+      labels:
+        - automation
+        - backport-skip
+
+sources:
+  beats:
+    kind: json
+    spec:
+      file: 'https://api.github.com/repos/elastic/beats/commits?sha={{ requiredEnv "BRANCH_NAME" }}&per_page=1'
+      key: '.[0].sha'
+
+
+conditions:
+  beats-version-check:
+    name: Check beats version
+    kind: shell
+    sourceid: beats
+    spec:
+      command: grep "BEATS_VERSION?={{ requiredEnv "BRANCH_NAME" }}" Makefile
+
+targets:
+  beats:
+    name: 'Update to elastic/beats@{{ source "beats" }}'
+    sourceid: beats
+    scmid: default
+    kind: shell
+    spec:
+      command: .ci/scripts/update-beats.sh
```

Will this work? so we can delete `.ci/scripts/update-beats.sh`?

```suggestion
      command: make update-beats BEATS_VERSION=
```

---

Unfortunately, no. The script is evaluates to 

`make update-beats BEATS_VERSION= xxxxx`

the whitespace makes it fail

---

Also tried variations like: 

```yaml
command: make update-beats BEATS_VERSION={{ source "beats" }}
```

but this evaluates to

```
make update-beats BEATS_VERSION=xxxx xxxx
```

and then it complains with

```
make: *** No rule to make target `xxxx`
```

---

Unfortunately, no: 

```
make: *** No rule to make target `#'.  Stop.
```

---

No worries, thanks for double checking it